spec/motion-spec/matchers/satisfy_spec.rb

```diff
@@ -0,0 +1,18 @@
+# -*- encoding : utf-8 -*-
+describe "Matcher::Satisfy" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.